lib/test/Test/Pos/Launcher/ConfigurationSpec.hs

```diff
@@ -5,22 +5,48 @@ module Test.Pos.Launcher.ConfigurationSpec
 import           Universum
 
 import           Data.Time.Clock.POSIX (getPOSIXTime)
-import           Test.Hspec (Spec, describe, it, shouldSatisfy)
+import           Test.Hspec (Spec, describe, it, shouldBe, shouldSatisfy)
 
+import           Ntp.Client (NtpConfiguration)
+import qualified Pos.Chain.Genesis as Genesis (Config (..))
+import           Pos.Chain.Txp (TxpConfiguration)
 import           Pos.Core.Slotting (Timestamp (..))
+import           Pos.Crypto (ProtocolMagic (..), RequiresNetworkMagic (..))
 import           Pos.Launcher.Configuration (ConfigurationOptions (..),
-                     defaultConfigurationOptions, withConfigurations)
+                     WalletConfiguration, defaultConfigurationOptions,
+                     withConfigurations)
 import           Pos.Util.Config (ConfigurationException)
 import           Pos.Util.Wlog (setupTestLogging)
 
+configFilePath :: FilePath
+configFilePath = "configuration.yaml"
```

Comment:
If we're lifting this out as a variable, should it be `"lib/configuration.yaml"`?

Reply:
Or perhaps [this line](https://github.com/input-output-hk/cardano-sl/pull/3715/files/ad5438cd6a7df6998c50df5aee0b205f68c47948#diff-ccfbc2ec33a789c8cb79c96adbca36c4R45) should add the `"lib/"` prefix